app/javascript/application.js

```diff
@@ -1,3 +1,29 @@
 // Entry point for the build script in your package.json
 import "@hotwired/turbo-rails"
 import "./controllers"
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Greeting from "./components/greeting";
+import { BrowserRouter, Route, Router } from "react-router-dom";
+import { Provider } from "react-redux";
+import store from "./store";
+
+const App = () => {
+  // add react router here
+  return (
+    <Provider store={store}>
+      <Greeting />
+      {/* <BrowserRouter> */}
+        {/* <Router> */}
+          {/* <Route path="/" component={<Greeting />} /> */}
+        {/* </Router> */}
+      {/* </BrowserRouter> */}
```

Comment:
- You have done great so far :+1: :+1: Your app meets all the requirements and works as expected. but there are a few things you need to correct before moving to the next milestone. :smile: :wave: 

- [ ] Following the best practices, you are not meant to leave unused code as a comment on your code base. it makes your codebase look unprofessional. I'll suggest you remove all these commented-out codes . :smile: :wave:

Reply:
thanks for pointing it out